README.md

```diff
@@ -375,6 +376,34 @@ export default async function () {
 }
 ```
 
+### Lambda
+
+Consult the `LambdaClient` [dedicated k6 documentation page](https://k6.io/docs/javascript-api/jslib/aws/lambdaclient) for more details on its methods and how to use it.
+
+```javascript
+import { AWSConfig, LambdaClient } from 'https://jslib.k6.io/aws/0.11.0/lambda.js'
```

Comment:
```suggestion
import { AWSConfig, LambdaClient } from 'https://jslib.k6.io/aws/0.12.0/lambda.js'
```
I expect we need to bump, right @oleiade?

Reply:
Yes, we should bump it. Some properties have been changed in a way that breaks the contract. The end result is more idiomatic JS and a good change, but on the off chance that someone started using the current version we should avoid breaking their code.